### PR TITLE
Xcode 9 support & adjustments

### PR DESCRIPTION
**Pull request by kafejo**
This pull request 
- Fixes XCode9 name identifier.
- Removes example methods.
- Merges Configurator into ViewController.
- Removes support for UITableViewController and UICollectionViewController (just create UIViewController based scene and change subclass).
- Improves overall naming conventions.

### CODE REVIEW

File: Clean Swift/Interactor.xctemplate/___FILEBASENAME___Interactor.swift [hunk 1]
```diff
@@ -8,24 +8,13 @@
...
 
-protocol ___FILEBASENAMEASIDENTIFIER___InteractorOutput {
-    func presentSomething(response: ___FILEBASENAMEASIDENTIFIER___Response)
+protocol ___VARIABLE_sceneName___InteractorOutput {
 }
 
-class ___FILEBASENAMEASIDENTIFIER___Interactor: ___FILEBASENAMEASIDENTIFIER___ViewControllerOutput {
-    var output: ___FILEBASENAMEASIDENTIFIER___InteractorOutput!
-    var worker: ___FILEBASENAMEASIDENTIFIER___WorkerProtocol!
+class ___VARIABLE_sceneName___Interactor: ___VARIABLE_sceneName___ViewControllerOutput {
```
Comment (mkj-is) [inline Clean Swift/Interactor.xctemplate/___FILEBASENAME___Interactor.swift]: We are currently using `final` keyword for classes by default. For most layers in the architecture it makes perfect sense. I am just writing it here as a suggestion.

File: Clean Swift/Interactor.xctemplate/___FILEBASENAME___Interactor.swift [hunk 2]
```diff
@@ -8,24 +8,13 @@
...
+    var presenter: ___VARIABLE_sceneName___InteractorOutput!
+    var worker: ___VARIABLE_sceneName___WorkerProtocol! = ___VARIABLE_sceneName___Worker()
```
Comment (mkj-is) [inline Clean Swift/Interactor.xctemplate/___FILEBASENAME___Interactor.swift]: The force-cast is not needed here, isn't it? Even for testing.

**Review by mkj-is — APPROVED** [review]
No problem, we are happy that you finally managed to do this!

File: Clean Swift/Interactor.xctemplate/___FILEBASENAME___Interactor.swift (hunk 2 above)
Comment (kafejo) [reply Clean Swift/Interactor.xctemplate/___FILEBASENAME___Interactor.swift]: It actually is needed. Usually, you want to supply your own worker spy that conforms to `___VARIABLE_sceneName___WorkerProtocol` in order to test whether worker's methods are called.

File: Clean Swift/Interactor.xctemplate/___FILEBASENAME___Interactor.swift (hunk 1 above)
Comment (kafejo) [reply Clean Swift/Interactor.xctemplate/___FILEBASENAME___Interactor.swift]: Good idea. Updated. 😊

**Review by mkj-is — APPROVED** [review]

**Comment by mkj-is** [comment]
@kafejo If you delete your fork, we can transfer the repository to you.

**Comment by kafejo** [comment]
Done. Thanks 🙂

**Pull request merged**